lib/badge-data.js

```diff
@@ -59,8 +63,10 @@ function makeLabel(defaultLabel, overrides) {
 }
 
 function makeLogo(defaultNamedLogo, overrides) {
-  const maybeDataUri = prependPrefix(overrides.logo, 'data:');
-  const maybeNamedLogo = overrides.logo === undefined ? defaultNamedLogo : overrides.logo;
+  const logo = 'logo' in overrides ? '' + overrides.logo : undefined;
```

Comment:
There could be a related issue when we index into the known logos, so seems safest to coerce here too.